kombu/transport/redis.py

```diff
@@ -779,7 +779,7 @@ def _size(self, queue):
     def _q_for_pri(self, queue, pri):
         pri = self.priority(pri)
         if pri:
-            return "{{{}}}{}{}".format(queue, self.sep, pri)
+            return "{}{}{}".format(queue, self.sep, pri)
```

Comment:
can you also update tests?

Reply:
Yep ofcourse, I've just been fighting with tox locally atm,